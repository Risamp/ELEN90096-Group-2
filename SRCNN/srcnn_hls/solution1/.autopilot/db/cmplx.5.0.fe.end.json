{
    "1": {
        "name": "conv1",
        "file": "",
        "fileloc": "0",
        "instructions": "399",
        "total_instructions": "399",
        "calls": "",
        "basicblocks": "47",
        "loads": {
            "G": "3",
            "L": "10"
        },
        "stores": {
            "G": "3",
            "L": "2"
        },
        "loops": {
            "L": "1",
            "N": "14"
        }
    },
    "0": {
        "name": "srcnn",
        "file": "",
        "fileloc": "0",
        "instructions": "226",
        "total_instructions": "625",
        "calls": "1",
        "basicblocks": "28",
        "loads": {
            "G": "3",
            "L": "2",
            "T": "4"
        },
        "stores": {
            "G": "1",
            "L": "4",
            "T": "1"
        },
        "loops": {
            "L": "2",
            "N": "7"
        }
    }
}
