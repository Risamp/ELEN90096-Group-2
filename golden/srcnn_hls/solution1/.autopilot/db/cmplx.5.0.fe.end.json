{
    "0": {
        "name": "srcnn",
        "file": "",
        "fileloc": "0",
        "instructions": "349",
        "total_instructions": "349",
        "calls": "",
        "basicblocks": "43",
        "loads": {
            "G": "4",
            "L": "3",
            "T": "7"
        },
        "stores": {
            "G": "2",
            "L": "6",
            "T": "1"
        },
        "loops": {
            "L": "3",
            "N": "11"
        }
    }
}
